### PR TITLE
Feat:Create blog section for ios development page

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
**##** Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

**Description:**
1.Create blog section.
**Video:**
https://github.com/canopas/canopas-website/assets/98794636/9fceaffc-2af6-421e-9aa1-552a4d798ca0
**Desktop Image**
image

**Mobile Image:**
image

**Desktop Performance :**
image
**Mobile Performance:**
Screenshot from 2023-07-06 16-16-08

### CODE REVIEW

File: vue-frontend/src/components/ios-app-development/BlogSection.vue [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        :modules="modules"
+      >
+        <swiper-slide
+          v-for="(blog, index) in blogs"
+          :key="index"
+          class="tw-mt-8"
+        >
+          <div
+            :class="blog.className"
+            class="tw-w-60 xs:tw-w-80 tw-h-[350px] tw-drop-shadow-md tw-text-left tw-cursor-pointer"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/ios-app-development/BlogSection.vue:49]: increase some card width in below of 420 screen width
Screenshot from 2023-09-22 17-19-23

Comment (cp-priyanka-g) [reply vue-frontend/src/components/ios-app-development/BlogSection.vue:49]: its cutting in below xs devices.tha's why i reduced . in real device iphone it was cutting.

File: vue-frontend/src/components/ios-app-development/BlogSection.vue [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+          >
+            <img
+              @click="openBlog(blog.link, 'tap_ios_blog_section')"
+              :src="blog.image"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/ios-app-development/BlogSection.vue:53]: in mobile section , image looking blur

File: vue-frontend/src/components/ios-app-development/BlogSection.vue (hunk 1 above)
Comment (cp-mansi-d) [inline vue-frontend/src/components/ios-app-development/BlogSection.vue:49]: - should card keep in center between screen width 420 to 467.

File: vue-frontend/src/components/ios-app-development/BlogSection.vue [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+          '2400': {
+            slidesPerView: 2.5,
+          },
+        }"
+      >
+        <swiper-slide
+          v-for="(blog, index) in iosblog"
+          :key="index"
+          class="tw-cursor-pointer tw-relative tw-flex tw-flex-col tw-pl-4 sm:tw-pl-8"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/ios-app-development/BlogSection.vue:115]: Can you set this card design like the second row in the last section of the contribution page, I think it's better than this.

eg. The first and last card should be cut equally, not just the last card, cut both cards equally.

File: vue-frontend/src/components/ios-app-development/BlogSection.vue (hunk 1 above)
Comment (cp-mansi-d) [reply vue-frontend/src/components/ios-app-development/BlogSection.vue]: Still it doesn't look centered at some widths

[Screencast from 09-27-23 15:12:17.webm](https://github.com/canopas/canopas-website/assets/115449373/7dc90454-6904-41f4-8e40-1b2409ae5c36)

**Review by cp-mansi-d — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**